tool/e2e-tests/test-packs/test-packs.ts

```diff
@@ -3,16 +3,25 @@ import {apiTimeout, defaultWaitTimeout} from "../helpers"
 import path from "path"
 import {sendPageTitle} from "../locators"
 
-export async function loadClinicalTestPack1Examples(driver: ThenableWebDriver): Promise<void> {
-  const testPackUpload = await getTestPackUpload(driver)
-  uploadTestPack(testPackUpload, "Full Prescriber Volume Pack.xlsx")
-  await loadPrescriptionsFromTestPack(driver)
-  await driver.wait(until.elementsLocated(sendPageTitle), apiTimeout)
+export async function loadClinicalFullPrescriberTestPack(driver: ThenableWebDriver): Promise<void> {
+  await loadTestPack(driver, "Full Prescriber Volume Pack.xlsx")
+}
+
+export async function loadSupplierTestPack(driver: ThenableWebDriver): Promise<void> {
+  await loadTestPack(driver, "Supplier 1 Test Pack.xlsx")
+}
+
+export async function loadPrescriptionTypeTestPack(driver: ThenableWebDriver): Promise<void> {
+  await loadTestPack(driver, "Prescription Types Test Pack.xlsx")
+}
+
+export async function loadPrescriptionTypesWithInvalidTypesTestPack(driver: ThenableWebDriver): Promise<void> {
```

Comment:
these function names are super useful and friendly to read - now tempted to rename the test packs themselves to match these names. approved anyway cause not a big deal